modules/tensor_mechanics/src/materials/ComputeReturnMappingStress.C

```diff
@@ -77,58 +77,90 @@ ComputeReturnMappingStress::updateQpStress(RankTwoTensor & strain_increment,
       << std::endl;
   }
 
-  // compute trial stress
-  RankTwoTensor stress_last_iteration = _elasticity_tensor[_qp] * (strain_increment + _elastic_strain_old[_qp]);
-
   RankTwoTensor inelastic_strain_increment, elastic_strain_increment;
   inelastic_strain_increment.zero();
 
-  Real equivalent_delta_stress, first_equivalent_delta_stress;
+  Real l2norm_delta_stress, first_l2norm_delta_stress;
   unsigned int counter = 0;
 
+  std::vector<RankTwoTensor> inelastic_strain_increment_new;
```

Comment:
Now that you don't need the old one anymore, I'd rename this to `inelastic_strain_increment`.